### DISCUSSION TITLE
About hardware specification for Carla + Apollo

### DISCUSSION THREAD

**Discussion by ghost** [guardstrikelab/carla_apollo_bridge#46] [Q&A]
Hi, 

I'm really excited to try this Carla + Apollo bridge and I'm planning to buy a new computer for that. However, I'm not sure what kind of specifications I should aim for; what I know is that both Carla and Apollo are very demanding.

If you're willing, can you describe :
1. Your computer specification ? (GPU, CPU, OS, and/or other information)
2. How the performance of Carla + Apollo has been in your computer ? (e.g. the framerates) 
3. What kind of upgrade do you think you should have in order for Carla + Apollo to runs faster ?

Hopefully this will not only help me, but other folks who are considering Carla + Apollo. Thanks a lot !

**Comment by XiaoFei9704** [comment]
Sorry it took so long to reply.
1. I run Apollo and Carla on two different machines. apollo: 3060/i7/ubuntu20.04/24GBmem; carla: 4070/i9/ubuntu20.04/64GBmem. Actually, a 3080 machine is enough to run Apollo and Carla. Memory should preferably be greater than 32GB.
2. A 3080 computer runs smoothly
3. If you want to make Carla and Apollo run smoother, of course, upgrading your graphics card is the best way
